doc/FAQ.md

```diff
@@ -71,3 +71,68 @@ Testsuite summary for mobject 0.7
 # ERROR: 0
 ============================================================================
 ```
+
+* How can I run IOR with Mobject using the RADOS backend?
+
+An [IOR backend for RADOS](https://github.com/hpc/ior/blob/main/src/aiori-RADOS.c)
+has been developed and available since IOR 3.2. However, this backend currently
+has to be modified slightly to work with Mobject.
+
+### Installing with Spack
+
+If using Spack with the `mochi-spack-packages` repo to manage Mobject and its
+dependencies, one should simply install IOR with the Mobject variant by doing:
+
+```bash
+$ spack install ior+mobject@develop
+```
+
+> **_NOTE:_** If using a spack environment, one may need to instead add the IOR
+> spec to their environment:
+>
+>     spack add ior+mobject@develop
+>     spack install
+
+### Installing manually
+
+If managing Mobject and its dependencies manually, one should first obtain
+the IOR source code:
+
+```bash
+$ git clone https://github.com/shanedsnyder/ior.git
+```
+
+> **_NOTE:_** This fork of IOR is currently being used until the IOR RADOS
```

Comment:
@shanedsnyder can you confirm if there is still something needed from your IOR repo to use it against the mobject/rados API (other than the header library changes)?   Just want to make sure that these instructions are still current.

Reply:
See this PR upstream: https://github.com/hpc/ior/pull/444

Once this is merged, we can update our Spack package and these docs to avoid needing to use my fork anymore.

Reply:
See #59 for corresponding updates to docs here.